docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -372,18 +372,6 @@ Interval at which the tablet manager tries to cleanup split tablets that are no
 
 Default: `60`
 
-##### --enable_automatic_tablet_splitting
```

Comment:
@hari90 , Many customers have been dabbling around with tablet splitting gflag for a couple of reasons. Please leave this gflag as is. Otherwise, it can result in more questions on slack/CEs on how to enable / disable.

Reply:
We do not want more customers to dabble with this now that it is an AutoFlag and enabled for everyone. Will continue the discussion on slack.